woowacrew-domain/src/main/java/woowacrew/github/domain/GithubCommit.java

```diff
@@ -16,6 +17,9 @@
     private LocalDate date;
     private Integer point;
```

Comment:
```suggestion
    private int point;
```

primitive type 대신 Wrapper class를 사용하신 이유가 있나요? 지난번에 리뷰했어야 했는데, 미안합니다.

Reply:
좋은 피드백 감사합니다!

primitive type은 null을 표현할 수 없고, 0으로 표현해야 하는데
결국에는 0도 값이기 때문에 되도록이면 Wrapper class를 사용하고 있습니다ㅎ

하지만 현재의 경우에는 point는 0으로 초기화를 시켜줘도 상관은 없을 것 같다는 생각이드네요..🤔